### ISSUE TITLE
3D plots are crashing the Python Console

### ISSUE THREAD

**Issue by otoonas** [spyder-ide/spyder#2556]
I'm trying to create a 3D line plot from a data file I have generated.
Using the following code:

```python
from mpl_toolkits.mplot3d import Axes3D
import matplotlib.pyplot as plt
import numpy as np

x = np.loadtxt("data.txt", usecols = (0,))
y = np.loadtxt("data.txt", usecols = (1,))
z = np.loadtxt("data.txt", usecols = (2,))

fig = plt.figure()
ax = fig.add_subplot(111, projection = '3d')
ax.plot(x, y, z)
plt.draw()
```

Rather than produce a figure, the error "python.exe has stopped working" flashes up, and the code crashes. I have tried it with both plt.draw() and plt.show(), and I'm using the Python console.

Any help is greatly appreciated!

## Version and main components

* Spyder Version: 2.3.5.2 
* Python Version: 2.7.10
* Qt Versions:  4.8.6, PyQt4 (API v2) 4.10.4 on Windows

## Optional dependencies
```
pyflakes >=0.5.0:  0.9.2 (OK)
pep8 >=0.6      :  1.6.2 (OK)
IPython >=1.0   :  3.2.0 (OK)
zmq >=2.1.11    :  14.7.0 (OK)
pygments >=1.6  :  2.0.2 (OK)
pandas >=0.13.1 :  0.16.2 (OK)
sphinx >=0.6.6  :  1.3.1 (OK)
rope >=0.9.2    :  0.9.4 (OK)
jedi >=0.8.1    :  0.8.1 (OK)
matplotlib >=1.0:  1.4.3 (OK)
sympy >=0.7.3   :  0.7.6 (OK)
pylint >=0.25   :  None (NOK)

```

**Comment by ccordoba12** [comment]
Closing as duplicate of issue #2402. Please see what I mentioned in that issue.

**Comment by ccordoba12** [comment]
Also, please the IPython console instead, it has much better support for this kind of things

**Comment by ccordoba12** [comment]
Sorry, I didn't noticed that your bug report is about 3d plots crashing the console.

**Comment by ccordoba12** [comment]
To test this, please give us a reproducible example, i.e. a sample code we can run in our side to test what's happening.

The code you posted is fine, but we don't have `data.txt` here :-)

**Comment by otoonas** [comment]
Yeah I tried everything in that thread, and it didn't work, hence this thread!

It's not letting me attach the original data file, so here's 20 lines from that txt file:

-1.72187E+11	26778156067	2993221937
-1.72187E+11	26778155661	2993221914
-1.72187E+11	26778152002	2993221704
-1.72187E+11	26778119073	2993219813
-1.72187E+11	26777822718	2993202797
-1.72186E+11	26775155518	2993049647
-1.72177E+11	26751150389	2991671260
-1.72095E+11	26535078034	2979262846
-1.71346E+11	24588349442	2867351307
-1.63438E+11	6935403396	1842514233
-1.51581E+11	-13326471994	641250479.9
-1.37144E+11	-32271033798	-512004943.5
-1.21865E+11	-47954652811	-1496637509
-1.01793E+11	-63766770271	-2531251878
-81504042137	-75507776508	-3347847617
-61847560744	-83516711285	-3957113660
-43343158610	-88351995100	-4384555424
-26656285599	-90583113792	-4654966919
-13041580332	-90925080555	-4795713948
-2416691624	-90254080582	-4854877882

**Comment by ccordoba12** [comment]
I'm sorry but I can't reproduce your bug, using your exact same configuration and the code you gave us.

Please run this command in a terminal (i.e. `cmd.exe`)

    spyder --reset

to reset Spyder settings and try again. Perhaps that will solve your problem.

**Comment by otoonas** [comment]
Yes that fixed it! Not sure how, as I tried resetting it using the .exe in the anaconda folder, and even uninstalled and reinstalled it before starting this thread! Thank you!

**Comment by ccordoba12** [comment]
Great! I'm glad this is solved for you :-)

**Comment by ChrisZeThird** [comment]
Hi, I also have a similar issue and reseting Spyder didn't work. I tried with a basic example provided by matplotlib documentation and it worked so it might be related to the size of the arrays I am working with. Is there a way to counter that, my pc isn't that bad after all?
